Resources/doc/filters.md

```diff
@@ -126,7 +126,7 @@ For instance, exclude entries with a property value of `null`, with the followin
 services:
     resource.date_filter:
         parent:    "api.doctrine.orm.date_filter"
-        arguments: [ { "dateProperty": ~ } ]
```

Comment:
The null value is for using database default (i.e. no special null handling). That has not been changed.

What I'm doing here is correcting the example for excluding null values (view the full file to see the text above it).